specs/topdown.md

```diff
@@ -0,0 +1,84 @@
+#Topdown Finality
+
+The topdown finality propagates the states in the parent subnet to the child blockchain. The following data are passed:
```

Comment:
I'd suggest to first describe what is the purpose of topdown finality in IPC, then when exactly it happens, and only then the exact data which is being passed from parent to the child. Wdyt?